moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_display.cpp

```diff
@@ -1032,25 +1018,24 @@ void MotionPlanningDisplay::changedPlanningGroup()
   if (!getRobotModel() || !robot_interaction_)
     return;
 
-  if (!planning_group_property_->getStdString().empty())
-    if (!getRobotModel()->hasJointModelGroup(planning_group_property_->getStdString()))
-    {
-      planning_group_property_->setStdString("");
-      return;
-    }
+  if (!planning_group_property_->getStdString().empty() &&
+      !getRobotModel()->hasJointModelGroup(planning_group_property_->getStdString()))
+  {
+    planning_group_property_->setStdString("");
+    return;
+  }
   modified_groups_.insert(planning_group_property_->getStdString());
 
-  if (robot_interaction_)
-    robot_interaction_->decideActiveComponents(planning_group_property_->getStdString());
+  robot_interaction_->decideActiveComponents(planning_group_property_->getStdString());
 
   updateQueryStartState();
   updateQueryGoalState();
   updateLinkColors();
 
   if (frame_)
     frame_->changePlanningGroup();
-  addBackgroundJob(boost::bind(&MotionPlanningDisplay::publishInteractiveMarkers, this, false),
-                   "publishInteractiveMarkers");
+  addBackgroundJob(boost::bind(&MotionPlanningDisplay::publishInteractiveMarkers, this, false), "publishInteractiveMark"
+                                                                                                "ers");
```

Comment:
Is this really according to the code style? It seems odd. =]

Reply:
Thanks for pointing this out. This has improved since  #1214.